packages/mui-material/src/Select/Select.test.js

```diff
@@ -393,6 +393,21 @@ describe('<Select />', () => {
     });
   });
 
+  it('should not have the selectable option selected when inital value provided is empty string on Select with ListSubHeader item', () => {
```

Comment:
An additional test should be added with just `MenuItem` components where the first item is disabled.